uportal-war/src/main/resources/properties/i18n/Messages.properties

```diff
@@ -391,6 +391,7 @@ most.frequently.added=Most Frequently Added
 move=Move
 move.this.column=Move this column
 move.this.portlet=Move this portlet
+move.this.portlet.error=This change was disallowed by the server.  The portlet will return to its original position.  Reload the page now?
```

Comment:
@timlevett -- James has the essence of it...
- The theme is already attempting to prevent illegal moves, and does a pretty good job.  As we identify cracks in that logic, we should address them as well.  I want to get this change in because I noticed that (a) the REST API was already sending 403 Forbidden whenever it detected an illegal move, but (b) the front end was not reacting to the non-200 HTTP status in any way.
- The portlet reorderer uses this fluid component:  https://wiki.fluidproject.org/display/docs/Layout+Reorderer+API.  My first choice would be to prevent/undo the move in these cases, but the event model of this widget does not make that easy.  (Widget only allows you to understand & deal with the move after it's done;  provides no "cancel" mechanism.) I don't we can put off other priorities long enough to make this fix that small amount better.

And actually -- this fix does "undo" the move, just at the cost of a page re-draw.